frontend/tests/e2e/search/features/work-flow-service-NegativeScenarios.feature

```diff
@@ -0,0 +1,149 @@
+
+        # Goal: What are the E2E tests (or other integrations) we should focus on building in Quad 4 that significantly reduces of our risk of breaking the Apply workflow unintentionally?
+
+        # In evaluating user interactions, build test cases for each scenario and rank them by their importance to the proper functioning of the apply functionality. Test cases can be stored, for now, in google drive document or spreadsheet until we have a better place to put them
+
+        # Acceptance criteria
+
+        # Primary apply related user flows are documented as test cases and ranked in order of priority for testing
+  ###########################################################################
+
+  ###########################################################################
+  # 1. Start Application – Shell Fails to Load
+  ###########################################################################
+  @negative @workflow @start
+  Scenario: Start Application fails to load Apply shell
+    Given the user is on the landing page
+    When the user selects "Start Application"
+    Then the shell fails to load
+    And the user sees an error message "Unable to load application"
+
+  ###########################################################################
+  # 2. Form Navigation – Data Lost or Fails to Navigate
+  ###########################################################################
+  @negative @workflow @navigation
+  Scenario: Form navigation loses data after Back and Next
+    Given the user is on Step 2 and enters valid information
+    When the user clicks Next and then Back
+    Then the previously entered data is missing
+    And the user sees a warning about lost progress
+
+  ###########################################################################
+  # 3. Required Field Validation – Empty Fields Block Progress
+  ###########################################################################
+  @negative @validation @workflow
+  Scenario: User attempts to proceed with empty required fields
+    Given the user is on a form step with required fields
+    When the user clicks Next without entering required information
+    Then progress is blocked
+    And error messages appear for each required field
+
+  ###########################################################################
+  # 4. Cross Form Required Validation – Missing Required Data at Submission
+  ###########################################################################
+  @negative @validation @submission @workflow
+  Scenario: Submission fails due to missing answers from previous steps
+    Given the user skipped a required question in an earlier step
+    When the user clicks Submit on the Review & Submit page
```

Comment:
there isn't a review and submit page, the submit button is on the main application page